src/libs/actions/App.js

```diff
@@ -45,28 +45,22 @@ Onyx.connect({
     },
 });
 
-let policyIDList = [];
-Onyx.connect({
-    key: ONYXKEYS.COLLECTION.POLICY,
-    waitForCollectionCallback: true,
-    callback: (policies) => {
-        policyIDList = _.compact(_.pluck(policies, 'id'));
-    },
-});
-
 // When we reconnect the app, we don't want to fetch workspaces created optimistically while offline since they don't exist yet in the back-end.
 // If we fetched them then they'd return as empty objects which would clear out the optimistic policy values initially created locally.
 // Once the re-queued call to CreateWorkspace returns, the full contents of the workspace excluded here should be correctly saved into Onyx.
 let policyIDListExcludingWorkspacesCreatedOffline = [];
```

Comment:
NAB I know that this was introduced in a different PR, but I think this is part of a bigger issue (see this [thread](https://expensify.slack.com/archives/C03TQ48KC/p1666371883290389)). 

The original intent of the `policyIDList` param was to clean up locally stored policies in case the user missed the delete policy pusher event. I think once we solve the issue in the thread that I linked, we should clean this up and send all policy IDs like we were doing before.

Reply:
Ty for linking that, I read it and https://github.com/Expensify/Expensify/issues/229696. I understood that the problem is that read command can potentially be executed before queued offline writes. This caused flickering because there is optimistic stuff removed from onyx that gets re-added back later. I feel like this code here should be fixing that by omitting the optimistic policies, I'm not sure why it remains a problem.

I noticed that @jasperhuangg passed the policy ids with an incorrect parameter name, so the backend is completely ignoring the list: https://github.com/Expensify/App/pull/11130/files#r1004909597, could that be why it remains a problem?